deltachat/event.go

```diff
@@ -0,0 +1,424 @@
+package deltachat
+
+type eventType string
+
+const (
+	eventInfo                       eventType = "Info"
```

Comment:
I think it's a little confusing lowercase events are types and uppercase the actual structs. Would prefer that "type" is added to event type names now that there's a distinction, even when it's only internal.

Reply:
done, thanks for the feedback!